.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,7 +1,5 @@
 ---
-name: Bug report
-about: Create a report to help us improve
-title: ''
+name: Bug report about: Create a report to help us improve title: ''
```

Comment:
What happened here? This is unintended right?

Reply:
Yep..